index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
+[=triggered by user activation|user triggered activation=], may be used as a
+[=new information about the user's intent|criteria=] to grant access to [=sensor readings|readings=].
```

Comment:
We wrote the permissions spec to give browsers a giant heap of freedom to decide what's "information about user's intent", but if a browser owner asked me, I'd advise not to treat an arbitrary gesture as that kind of information, and I think specs shouldn't encourage it either.